src/alloca.h

```diff
@@ -0,0 +1,27 @@
+/*
+* Copyright 2016 WebAssembly Community Group participants
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#ifndef WASM_ALLOCA_H_
+#define WASM_ALLOCA_H_
+
+#ifdef __GNUC__
```

Comment:
Actually, rather than hiding a system header, could you move this definition and other platform-specific stuff to `wasm-platform.h`?